charts/kubernetes-external-secrets/crds/kubernetes-client.io_externalsecrets_crd.yaml

```diff
@@ -8,71 +8,73 @@ metadata:
     app.kubernetes.io/managed-by: helm
 spec:
   group: kubernetes-client.io
-  version: v1
   scope: Namespaced
 
-  names:
-    shortNames:
-      - es
-    kind: ExternalSecret
-    plural: externalsecrets
-    singular: externalsecret
-
-  additionalPrinterColumns:
-    - JSONPath: .status.lastSync
-      name: Last Sync
-      type: date
-    - JSONPath: .status.status
-      name: status
-      type: string
-    - JSONPath: .metadata.creationTimestamp
-      name: Age
-      type: date
+  preserveUnknownFields: false
 
-  validation:
-    openAPIV3Schema:
-      properties:
-        spec:
+  versions:
+    - name: v1
+      served: true
+      storage: true
+      subresources:
+        status: {}
+      schema:
+        openAPIV3Schema:
           type: object
+          required:
+            - spec
           properties:
-            template:
-              description: Template which will be deep merged without mutating
-                any existing fields. into generated secret, can be used to
-                set for example annotations or type on the generated secret
+            spec:
```

Comment:
we could set `x-kubernetes-preserve-unknown-fields: true` to allow extra fields here